templates/connected-base.html.twig

```diff
@@ -6,13 +6,10 @@
             <img class="header__logo" src="{{ asset('build/images/logo-demoapp.svg') }}"
                  alt="{{ 'page.welcome.connect.logo_alt' | trans }}" />
             <nav class="main-navigation">
-                <a href="{{ path('products') }}" class="main-navigation__link main-navigation__link--active" >
-                    {{ 'common.header.links.catalog' | trans }}
-                </a>
-                <a href="https://marketplace.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link" >
+                <a href="https://marketplace.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link main-navigation__link--external" >
```

Comment:
```suggestion
                <a href="https://apps.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link main-navigation__link--external" >
```

Reply:
Just to explain: marketplace was the old name of the "Akeneo app store" so this hostname is deprecated.